exchanges/okx/okx_websocket.go

```diff
@@ -1232,38 +1232,41 @@ func (ok *Okx) wsProcessTickers(data []byte) error {
 func (ok *Okx) GenerateDefaultSubscriptions() ([]stream.ChannelSubscription, error) {
 	var subscriptions []stream.ChannelSubscription
 	assets := ok.GetAssetTypes(true)
+	var subs []string
+	copy(subs, defaultSubscribedChannels)
```

Comment:
This still isn't correct as copy doesn't allocate memory resulting in an empty subs, you can instead do something like this:

```
subs := make([]string, 0, len(defaultSubscribedChannels) + 2)
subs = append(subs, defaultSubscribedChannels...)
if ok.Websocket.CanUseAuthenticatedEndpoints() {
    subs = append(subs, okxChannelAccount, okxChannelOrders)
}
```